wpsc-components/fancy-notifications/js/fancy-notifications.js

```diff
@@ -0,0 +1,88 @@
+
+/**
+ * WP eCommerce Fancy Notifications JS
+ */
+
+var WPEC_Fancy_Notifications;
+
+( function( $ ) {
+
+	WPEC_Fancy_Notifications = {
+
+		/**
+		 * Move Fancy Notification element to end of HTML body.
+		 */
+		appendToBody : function() {
+
+			$( '#fancy_notification' ).appendTo( 'body' );
+
+		},
+
+		/**
+		 * Fancy Notification: Show
+		 */
+		wpscAddToCart : function( e ) {
+
+			$( 'div.wpsc_loading_animation' ).css( 'visibility', 'hidden' );
+			WPEC_Fancy_Notifications.fancy_notification( e.form );
+
+		},
+
+		/**
+		 * Fancy Notification: Hide
+		 */
+		wpscAddedToCart : function( e ) {
+
+			if ( ( e.response ) ) {
+				if ( e.response.hasOwnProperty( 'fancy_notification' ) && e.response.fancy_notification ) {
+					if ( $( '#fancy_notification_content' ) ) {
+						$( '#fancy_notification_content' ).html( e.response.fancy_notification );
+						$( '#loading_animation').css( 'display', 'none' );
+						$( '#fancy_notification_content' ).css( 'display', 'block' );
+					}
+				}
+				$( document ).trigger( { type : 'wpsc_fancy_notification', response : e.response } );
+			}
+
+			if ( $( '#fancy_notification' ).length > 0 ) {
+				$( '#loading_animation' ).css( 'display', 'none' );
+			}
+
+		},
+
+		/**
+		 * Fancy Notification
+		 *
+		 * @param  object  parent_form  Form element.
+		 */
+		fancy_notification : function( parent_form ) {
+
+			var fancyNotificationEl = $( '#fancy_notification' );
+
+			if ( typeof( WPSC_SHOW_FANCY_NOTIFICATION ) == 'undefined' ) {
+				WPSC_SHOW_FANCY_NOTIFICATION = true;
```

Comment:
Also from Scrutinizer: `WPSC_SHOW_FANCY_NOTIFICATION does not seem to be defined.`